doc/source/whatsnew/v0.24.2.rst

```diff
@@ -52,7 +52,7 @@ Bug Fixes
 **I/O**
 
 - Bug in reading a HDF5 table-format ``DataFrame`` created in Python 2, in Python 3 (:issue:`24925`)
--
+- Bug in retaining timezone in DateTimeIndex when saving and loading the DataFrame from msgpack. (:issue:`25148`)
```

Comment:
Double back ticks around msgpack. Single backticks around DatetimeIndex and DataFrame with `:class:` before each